common/defs-cmds.go

```diff
@@ -539,4 +539,13 @@ const StartDevUsageText = `The `+"`"+`temporal server start-dev`+"`"+` command s
 The results of any command run on the Server can be viewed at http://localhost:7233.
 `
 
-const CompletionText = "testing something"
\ No newline at end of file
+const CustomTemplateHelpCLI = `NAME:
```

Comment:
What does `CLI` in the variable name stand for?

Reply:
I needed to make the variable name stand out.